src/database/Database.cpp

```diff
@@ -212,31 +212,6 @@ Database::applySchemaUpgrade(unsigned long vers)
     soci::transaction tx(mSession);
     switch (vers)
     {
-    case 14:
-        mApp.getPersistentState().setRebuildForType(OFFER);
-        break;
-    case 15:
-        mApp.getPersistentState().setRebuildForType(TRUSTLINE);
-        mApp.getPersistentState().setRebuildForType(LIQUIDITY_POOL);
-        break;
-    case 16:
-        mApp.getPersistentState().setRebuildForType(LIQUIDITY_POOL);
-        break;
-    case 17:
-        mApp.getPersistentState().setRebuildForType(OFFER);
-        break;
-    case 18:
-        createTxSetHistoryTable(*this);
-        mApp.getPersistentState().upgradeSCPDataFormat();
-        break;
-    case 19:
-        mApp.getPersistentState().upgradeSCPDataV1Format();
```

Comment:
There's a character limit schema upgrade in `upgradeSCPDataV1Format` that needs to be incorporated into core's startup path now that we removed this code. That being said, how come none of the tests caught it?

Reply:
+1 also we should delete that `upgradeSizeLimit` method and just change the create table statement right?

Reply:
Yeah, precisely

Reply:
I'm not sure why the tests passed, how do you expect them to fail? Do we write a statename greater than 32 characters somewhere?

Reply:
SCP data in this format is always greater than 32 characters, because we write tx set hashes. SQLite doesn't actually enforce character limit, but I'd expect Postgres tests to fail, e.g. `SCP State` in HerderTests

Reply:
Ah I see, "SCP State" is an acceptance test, which we don't run in Github CI IIRC. Maybe we should remove the acceptance tag on this test: it's cheap to run and it's important for testing persistence/core restarts

Reply:
I changed removed the acceptance tag on SCP State in the Test case declaration. It passes with postgres on my mac, I just pushed some refs so we'll see if it passes on CI.

Reply:
Actually I changed the create statement in the same diff, which is why its passing.